CI/check_pragma_once.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+ec=0
+
+for file in $(find Core Examples Tests Plugins -name "*.hpp"); do
+    res=$(grep "#pragma once" $file)
```

Comment:
maybe pedantic (feel free to ignore) but you make this more robust, e.g. `grep -e '^[[:space:]]*#pragma once' test.cxx` . This way you will catch instance where the pragma is commented-out for instance

Reply:
nice, I changed the `grep` command!